tooling-tests/src/test/java/io/ballerina/stdlib/grpc/tools/ToolingCommonTest.java

```diff
@@ -185,6 +185,13 @@ public void testDuplicateOutputType() {
                 "duplicate_output_type_pb.bal", "tool_test_data_type_22");
     }
 
+    @Test
+    public void testHelloWorldWithReservedKeywords() {
```

Comment:
Btw, isnt it good to rename the test case and the resource file since there is a keyword test already `testHelloWorldWithReservedNames`.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/702/commits/d57f0988443ca5f94f293269ef1e4b6bc0021f75